diktat-analysis.yml

```diff
@@ -2,9 +2,11 @@
   enabled: true
   configuration:
     domainName: org.cqfn.save
-    kotlinVersion: 1.5
+    kotlinVersion: 1.6
     srcDirectories: "main,nativeMain,commonNonJsMain"
     testDirs: "test,nativeTest,commonTest,jvmTest,commonNonJsTest"
+- name: AVOID_NULL_CHECKS
+  enabled: false
```

Comment:
May be some link to https://github.com/analysis-dev/diktat/issues/1167?

Reply:
It's not specifically because of issues, I just think that generally this rule overcomplicates code in SAVE, where we have a lot of branching. So simple comparisons with null look more clear. I think, we discussed this some time ago

Reply:
ok